src/main/java/com/crowdin/cli/commands/picocli/CommandNames.java

```diff
@@ -52,4 +52,10 @@ public final class CommandNames {
     public static final String UPLOAD_TRANSLATIONS = "translations";
 
     public static final String PRE_TRANSLATE = "pre-translate";
+
```

Comment:
please group these variables like the commands above

Reply:
done